### ISSUE TITLE
workspace issue

### ISSUE THREAD

**Issue by jimdavis1** [PATRIC3/patric3_website#1923]
1.  I have no idea how many items are in this workspace. It would be useful to know from the website.  I do know that there are >5000 jobs.

2.  I need to find "SRR3664859" and the only way to do that from my workspace page is by sorting and scrolling.

3. I have 2 versions of this private genome. From which workspace directory  did they originate? This should exist on the genome page (see second picture).

4. I have had to reannotate these genomes twice, and will probably have to do it again a 3rd time before it gets made public because there is no "update annotation" button.   I will jam them back into the queue in a day or two.  Sooner or later, this will begin to interfere with new annotation jobs.

![image](https://user-images.githubusercontent.com/7661533/37717382-71784e04-2cee-11e8-9193-e99744844211.png)

![image](https://user-images.githubusercontent.com/7661533/37717603-dd64e6d6-2cee-11e8-90e6-4e103229c1e6.png)

**Comment by dawenx** [comment]
@nconrad I hope I have re-assigned this to the right person

**Comment by rkenyon** [comment]
Jim triage comment: There are actually four bugs wrapped up in this ticket. 
#1.  Medium
#2.  Medium
#3.  High
#4.  High
This user account showing all of the problems that that are occurring once you have assembled more than 50 or so genomes.   

As a matter of fact, Maulik and I actually encountered #3 yesterday.  You have no path from a private genome on the website back to the job directory from which came.

**Comment by nconrad** [comment]
1: This is covered in another ticket: https://github.com/PATRIC3/patric3_website/issues/1839 . I'm adding the count either way.

2: Search isn't supported by the workspace service.  Note that using the usual browser search (cmd-f) gets you half way there for this particular scenario. 

3: Yeah, this is larger issue that would need team discussion.

4:  Bob was working on a CLI command to re-run jobs.  The plan was to add it to the UI afterwards, IIRC.

In summary, 2-4 are not currently possible.

**Comment by olsonanl** [comment]
@rkenyon shouldn't this get burst into separate issues for the four distinct problems?

**Comment by rkenyon** [comment]
Bursted into 4 tickets: #1839  #2083  #2084  #2085 

Best guesses for assignees.

**Comment by nconrad** [comment]
I'll close this since there are separate tickets.   Thanks, @rkenyon.
